File: py/99_bottles_of_beer.py
#!/usr/bin/env python

for i in range(99, 0, -1):
  print("%d bottle%s of beer on the wall" % (i, '' if i == 1 else 's'))
  print("%d bottle%s of beer" % (i, '' if i == 1 else 's'))
  print("Take one down, pass it around")
  print("%d bottle%s of beer on the wall" % (i - 1, '' if i - 1 == 1 else 's'))

File: pl/99_bottles_of_beer.pl
#!/usr/bin/env perl

my $x = 99;

my $beer = "bottles of beer";
my $full = "$beer on the wall\n";
for ( reverse 1 .. $x ) {
    print "$_ $full";
    print "$_ $beer\n";
    print "Take one down, pass it around\n";
    print $_-1 . " $full";
}
